dnf-plugins/py3query.py

```diff
@@ -289,9 +289,7 @@ def key(bug):
                                 .append(BUGZILLA_BUG_URL.format(tb))
 
                 if any(tb in bug.blocks for tb in MISPACKAGED_TRACKER_BUG_IDS):
-                    if r.get('status') == 'idle' and bug.status != 'NEW':
-                        r['status'] = 'in-progress'
-                    elif r.get('status') == 'idle' and bug.status == 'NEW':
+                    if r.get('status') == 'idle' and bug.status == 'NEW':
```

Comment:
We probably no longer need to check if `bug.status == 'NEW'`, because all the packages that were displayed as `in-progress` will remain `idle`.